src/Appender/FileAppender.cs

```diff
@@ -846,11 +851,15 @@ public override void ActivateOptions()
 							.Replace(":", "_")
 							.Replace("/", "_");
 
-					m_mutex = new Mutex(false, mutexFriendlyFilename);
+					m_mutexName = mutexFriendlyFilename;
```

Comment:
Storing this as an internal state appears to be unnecessary, isn't it?

Reply:
I'll remove it. See the previous comment above.